src/toil/test/sort/sort.py

```diff
@@ -33,8 +33,8 @@ def setup(job, inputFile, N, downCheckpoints):
     #Write the input file to the file store
     inputFileStoreID = job.fileStore.writeGlobalFile(inputFile, True)
     job.fileStore.logToMaster(" Starting the merge sort ")
-    job.addFollowOnJobFn(cleanup, job.addChildJobFn(down, 
```

Comment:
Try to eliminate this insignificant whitespace change. It makes for a noisy diff.